pkg/cmd/issue/comment/comment.go

```diff
@@ -37,15 +38,41 @@ func NewCmdComment(f *cmdutil.Factory, runF func(*prShared.CommentableOptions) e
 		Args: cobra.ExactArgs(1),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			opts.RetrieveCommentable = func() (prShared.Commentable, ghrepo.Interface, error) {
+				// TODO wm: more testing
```

Comment:
This block is totally untested. Not sure how much I want to tackle this for the extra confidence. TBD.